linchpin/cli/__init__.py

```diff
@@ -703,7 +707,8 @@ def _execute(self, provision_data, targets,
         return self.do_action(prov_data,
                               action=action,
                               run_id=run_id,
-                              tx_id=tx_id)
+                              tx_id=tx_id,
+                              vault_pass=vault_pass)
```

Comment:
See the reference above for `do_action`. `vault_pass` should not need to be passed, use the context object instead.